src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/data/batch-pool-create-invalid.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "azure-cli-test-json",
+  "vmSize": "small",
+  "cloudServiceConfiguration": {
+    "osFamily": "4",
+    "targetOSVersion": "*"
+  }
+  "targetDedicated": 2,
```

Comment:
This character is highlighted in red on GitHub which makes me suspicious.
Can you verify that it's a `:` (colon)?